sda-commons-server-openapi-example/README.md

```diff
@@ -5,7 +5,7 @@ This example module shows an
 [`OpenApiBundle`](../sda-commons-server-openapi/src/main/java/org/sdase/commons/server/openapi/OpenApiBundle.java)
 to describe REST endpoints with a OpenApi 3 documentation.
 
-Beside the initialization of the bundle via the [`SdaPlatformBundle`](../sda-commons-server-starter/src/main/java/org/sdase/commons/server/starter/SdaPlatformBundle.java),
+Beside the initialization of the bundle via the [`SdaPlatformBundle`](../sda-commons-server-starter-swagger/src/main/java/org/sdase/commons/server/starter/SdaPlatformBundle.java),
```

Comment:
I think the new OpenAPI code should reference the new bundle

Reply:
yes, done